autopush-common/src/db/bigtable/bigtable_client/mod.rs

```diff
@@ -1033,9 +1022,8 @@ impl DbClient for BigTableClientImpl {
         };
 
         row.cells.insert(
-            MESSAGE_FAMILY.to_owned(),
+            ROUTER_FAMILY.to_owned(),
```

Comment:
When we are [generating this value](https://github.com/mozilla-services/autopush-rs/blob/763e2afb3c87b06429868b09c0a365021e4cf2f5/autopush-common/src/db/bigtable/bigtable_client/mod.rs#L461-L462) in `user_to_row`, it's in the `router` family.